### ISSUE TITLE
Accessibility - Loading layers from file paths failed

### ISSUE THREAD

**Issue by mvmaltitz** [worldbank/GEEST#38]
Loading layers from file paths failed, and the processing did not work (see WTP example):

![Image](https://github.com/user-attachments/assets/bdf8be1d-8196-4503-8b9e-104b2aa4afd4)

When attempting to discard the failed processing and restart by loading the input data using the Clear Item button, nothing happens. The data must be discarded manually for each indicator.
When data is loaded from layers, the processing works.

**End Date:** 2026-04-09
